autoload/SpaceVim/plugins.vim

```diff
@@ -21,16 +20,18 @@ function! s:load_plugins() abort
   for group in SpaceVim#layers#get()
     let g:_spacevim_plugin_layer = group
     for plugin in s:getLayerPlugins(group)
-      if len(plugin) == 2
-        call zvim#plug#add(plugin[0], plugin[1])
-        if zvim#plug#tap(split(plugin[0], '/')[-1]) && get(plugin[1], 'loadconf', 0 )
-          call zvim#plug#defind_hooks(split(plugin[0], '/')[-1])
+      if index(g:spacevim_disabled_plugins, split(plugin[0], '/')[-1]) == -1
+        if len(plugin) == 2
```

Comment:
you should check if len(plugin) == 2 first, otherwise plugin[0] maybe cause error.

Reply:
I have changed the commit, pls check.